packages/node_modules/@ciscospark/internal-plugin-presence/src/presence.js

```diff
@@ -0,0 +1,127 @@
+/**!
+ *
+ * Copyright (c) 2015-2017 Cisco Systems, Inc. See LICENSE file.
+ * @private
+ */
+import '@ciscospark/internal-plugin-wdm';
+
+import {SparkPlugin} from '@ciscospark/spark-core';
+
+const defaultSubscriptionTtl = 600;
+
+/**
+ * @class
+ * @extends SparkPlugin
+ */
+const Presence = SparkPlugin.extend({
+  namespace: `Presence`,
+
+  /**
+   * Gets the current presence status of a given person or group of person ids
+   * @param {string | Array} personIds
+   * @returns {Promise}
+   */
+  get(personIds) {
+    let subjects;
+    if (!personIds) {
+      return Promise.reject(new Error(`A person id is required`));
+    }
+    if (Array.isArray(personIds)) {
+      subjects = personIds;
+    }
+    else {
+      subjects = [personIds];
+    }
+    return this.spark.request({
+      method: `POST`,
+      service: `apheleia`,
+      resource: `compositions`,
+      body: {subjects}
+    })
+    .then((response) => response.body);
```

Comment:
based on the transform above, won't this require the consumer to then lookup the result for a single user in the response? maybe we need a `get()` and a `list()`...